pkg/virt-controller/watch/workload-updater/workload-updater_test.go

```diff
@@ -437,35 +497,33 @@ var _ = Describe("Workload Updater", func() {
 
 	Context("LiveUpdate features", func() {
 		It("VMI needs to be migrated when memory hotplug is requested", func() {
-			vmi := api.NewMinimalVMI("testvm")
-
 			condition := v1.VirtualMachineInstanceCondition{
 				Type:   v1.VirtualMachineInstanceMemoryChange,
 				Status: k8sv1.ConditionTrue,
 			}
+			vmi := libvmi.New(
+				libvmi.WithName("testvm"),
+				libvmistatus.WithCondition(condition),
+			)
+
 			virtcontroller.NewVirtualMachineInstanceConditionManager().UpdateCondition(vmi, &condition)
 
 			Expect(controller.doesRequireMigration(vmi)).To(BeTrue())
 		})
 	})
 
 	Context("Abort changes due to an automated live update", func() {
-		createVM := func(annotation, hasChangeCondition bool) *v1.VirtualMachineInstance {
-			vmi := api.NewMinimalVMI("testvm")
-			vmi.Namespace = k8sv1.NamespaceDefault
-			vmi.Status.Phase = v1.Running
-			vmi.Status.Conditions = append(vmi.Status.Conditions, v1.VirtualMachineInstanceCondition{
-				Type: v1.VirtualMachineInstanceIsMigratable, Status: k8sv1.ConditionTrue})
-			if hasChangeCondition {
-				vmi.Status.Conditions = append(vmi.Status.Conditions, v1.VirtualMachineInstanceCondition{
-					Type:   v1.VirtualMachineInstanceMemoryChange,
-					Status: k8sv1.ConditionTrue,
-				})
-			}
-			if annotation {
-				vmi.ObjectMeta.Annotations = make(map[string]string)
-				vmi.ObjectMeta.Annotations[v1.WorkloadUpdateMigrationAbortionAnnotation] = ""
+		createVM := func(optionalOpts ...libvmi.Option) *v1.VirtualMachineInstance {
```

Comment:
Personal and controversial opinion:
This is a very local and purpose-built function created to avoid test code duplication.
To me, the previous function made more sense (`func(annotation, hasChangeCondition bool)`). These are the two things we care about in the many tests that follow (with/without "the" annotation, with/without "the" condition). It also avoids more code duplication.
I know some people don't like functions that take booleans, and for them we can just declare some dull consts (withAnnotation = true, withoutAnnotation = false, ...).
WDYT?

Reply:
Makes sense! Thank you